.cruft.json

```
@@ -1,13 +1,13 @@
 {
   "template": "https://github.com/projectsyn/commodore-component-template.git",
-  "commit": "f12cbb49f928cf689cf39b98d8d7c9e88fb289f6",
+  "commit": "af2d7732a95c20935b3579f2957006d539f9a619",
```

This should be `f12cbb49f928cf689cf39b98d8d7c9e88fb289f6` since that's what we've updated the latest master branch to:

```suggestion
  "commit": "f12cbb49f928cf689cf39b98d8d7c9e88fb289f6",
```